test/NosCore.Tests/BazaarTests/CRegPacketHandlerTests.cs

```diff
@@ -53,21 +55,21 @@ public class CRegPacketHandlerTest
     {
         private Mock<IBazaarHttpClient>? _bazaarHttpClient;
         private CRegPacketHandler? _cregPacketHandler;
-        private Mock<IGenericDao<InventoryItemInstanceDto>>? _inventoryItemInstanceDao;
-        private Mock<IGenericDao<IItemInstanceDto>>? _itemInstanceDao;
+        private Mock<IDao<InventoryItemInstanceDto, Guid>>? _inventoryItemInstanceDao;
+        private Mock<IDao<IItemInstanceDto?, Guid>>? _itemInstanceDao;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_itemInstanceDao' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)